pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!-- 
+Hello Axolotl!
+Thank you for contributing to the lakeFS project.
+We appreciate the time invested in this pull request and created this template to help make this process easier.
+It's really important to have all the information and context, to ensure we can properly address this PR 
+Please use the following references to fill out the pull request.
+--> 
+
+### Linked Issue
+
+Closes #(Insert issue number closed by this PR)
+
+---
+
+## Change Description
+
```

Comment:
We may want to add a background section to share context relevant to understanding the pr